README.md

```diff
@@ -46,15 +46,22 @@ If you would not, these installation are not required.
 
 ### Bootstraping container filesystem
 
+Create mount points:
+
+```sh
+$ mkdir -p /var/another/root/etc
+$ mkdir -p /var/another/root/home
+```
+
 Create the file `example.haco`:
 
 ```ruby
-Haconiwa::Base.define do |config|
```

Comment:
Got error as `Direct call of Haconiwa::Base.define is deprecated. Please rewrite into Haconiwa.define (RuntimeError)`